acceptance-tests/lib/testState.ts

```diff
@@ -6,15 +6,20 @@ import { existsSync, readFileSync, writeFileSync } from 'fs';
 import yaml from 'js-yaml';
 import rimraf from 'rimraf';
 import { v4 as uuidv4 } from 'uuid';
+import * as path from 'node:path';
+
+export const testOutputDir = 'test-output';
 
 export class TestState {
   public config: TestConfig;
   public cli: CLI;
   private testConfigFileName: string;
   private parsedYaml: ReturnType<yaml.load>;
+  private testOutputDir = 'test-output';
```

Comment:
Should this use the variable defined above? Just in case we ever change it

Reply:
Whoops, yep.  Good catch

Reply:
https://github.com/HubSpot/hubspot-cli/pull/1198/commits/e7b4c3d95d8a072afc89a652ff7853e8fd62a9c3